.nvmrc

```diff
@@ -1 +1 @@
-v16
+v18.6.0
```

Comment:
Is there a way to allow >=18.6.0? 🤔

Reply:
There's not much doc about `.nvmrc`. All that I've found was fixed versions

Reply:
I added the enforcement of `>=18.6.0` in `package.json` in a75f4ec but there is no way to specify a version range for `.nvmrc`